lib/dor/text_extraction/file_fetcher.rb

```diff
@@ -80,6 +79,14 @@ def fetch_and_write_file_to_disk(filename:, path:)
       def preservation_client
         @preservation_client ||= Preservation::Client.configure(url: Settings.preservation_catalog.url, token: Settings.preservation_catalog.token)
       end
+
+      def aws_client
+        Aws.config.update({
```

Comment:
i wonder if we could do this configuration in a more centralized way just once, since this is the global config, but i'm blanking on a specific way in the absence of the rails `config/initializers` feature.

Reply:
moved to its own AwsProvider class